assets/src/edit-story/components/panels/alignment/alignment.js

```diff
@@ -104,7 +108,18 @@ const alignmentButtonIds = [
 
 function ElementAlignmentPanel({ selectedElements, pushUpdate }) {
   const { isRTL } = useConfig();
-  const boundRect = getBoundRect(selectedElements);
+  // Set boundRect with pageSize when there is only element selected
+  const boundRect =
+    selectedElements.length === 1
+      ? {
+          startX: 0,
+          startY: 0,
+          endX: DEFAULT_EDITOR_PAGE_WIDTH,
+          endY: DEFAULT_EDITOR_PAGE_HEIGHT,
+          width: DEFAULT_EDITOR_PAGE_WIDTH,
+          height: DEFAULT_EDITOR_PAGE_HEIGHT,
+        }
```

Comment:
Maybe move this object to a `const PAGE_RECT = {};` or similar outside the component?

Reply:
Defined const object on top. But I think we don't need this declaration on something else for common.